### ISSUE TITLE
error comiling code

### ISSUE THREAD

**Issue by LasDes** [gipert/mugraphy#2]
hello I tried to compile the code in order to reporduce the findings in the paper, but getting the following error:

-------- EEEE ------- G4Exception-START -------- EEEE -------
*** G4Exception : InvalidUnit
      issued by : G4UnitDefinition::GetCategory()
The unit 'GeV/c' does not exist in the Units Table!
*** Fatal Exception *** core dump ***
 **** Track information is not available at this moment
 **** Step information is not available at this moment

-------- EEEE -------- G4Exception-END --------- EEEE -------

*** G4Exception: Aborting execution ***

**Comment by gipert** [comment]
Hi! Might be that you are using an incompatible G4 version. Which one is this? I've been mainly working with version 11.

If you have familiarity with Docker/Singularity containers, I would highly recommend making use of this container here: https://hub.docker.com/r/gipert/remage-base

**Comment by LasDes** [comment]
I have tried compiling it with Geant4 10.7

**Comment by gipert** [comment]
Then yes, you'll need to upgrade to version 11 because I specifically use those G4 features. I realized I forgot to bump the version requirement in `CMakeLists.txt` (fixed now in https://github.com/gipert/mugraphy/commit/77ce2050acf377f97bb88636ce67296409158c3d).

Can I ask you what's your use case? Be warned that the project is work in progress and it cannot reproduce the results of the original paper yet. I suspect there is a problem in the simulation but I did not manage to figure it out yet. Another problem is that the simulation is particularly inefficient (and therefore quite computationally intensive).
